_posts/2018-07-31-why-emacs-is-a-great-editor.md

```diff
@@ -0,0 +1,49 @@
+---
+layout:     post
+title:      Why Emacs is a great text editor
+date:       2018-07-31
+summary:    My thoughts on Emacs
+categories: emacs
+---
+
+![swiss-knife](https://i.imgur.com/J5Od0Xm.png)
+
+Often arguments arise from the fact that I use emacs as my main text editor. In this post, I'll try to explain why I think emacs is a great text editor and how it helps me be more productive every day.
+
+### History
+
+Before using emacs I've tried different text editors (Atom, Brackets etc) but I've primarily used Sublime for a couple of years before switching to emacs. There are two fundamental reasons that motivated me to learn it.
+
+The first thought that I should level up my text editor usage skills came to me when I was pair programming with my colleague solving a task at work. He uses Vim as his text editor. And in his turns he used his time much more productively than me navigating only with shortcuts, also he had shortcuts for every action he used. This experience was mind-blowing for me, he was like code wizard master compared to me navigating using a touchpad.
+
+The reason why I chose to learn emacs and not vim is that in that period of time in Elixir community emacs was very popular (and still is) and people were talking about it in podcasts and blogs. Also, there are some good plugins for Emacs for writing Elixir (alchemist, elixir-mode).
```

Comment:
Not sure if you're interested in this, but you could link to the repositories for `alchemist` and `elixir-mode` if you know them.